### ISSUE TITLE
JAWS issue when switching back to Thorium from another app

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#1112]
I am a person with vision impairment using Thorium Reader to read ePub
books with the assistance of a screen reader 'JAWS for Windows'.
Whenever I open aePub book in the Thorium Reader my screen reader
reads the book very well. However, whenever I switch to any other
application with 'Alt+Tab' command and return back to the Thorium
Reader JAWS doesn't reads the content of the book; only the title bar
of the Thorium Reader application can be read.
Therefore, I earnestly request your kind office to rectify the issue
at the earliest.

Thanks & Regards
Harshit Lohia

**Comment by sukiletxe** [comment]
This happens with NVDA too. A workaround is to press alt+space, then select any option. The book should be visible after that again. Weirdly it happens randomly to me.

**Comment by danielweck** [comment]
I am closing this issue for now as it is difficult to reproduce, and Thorium has since evolved with new versions of Electron and Chromium which might have improved the accessibility of embedded webviews. Feel free to reopen, hopefully with a detailed reproducible usage pattern so that we can pin-point the exact conditions that trigger the screen reader buffer refresh bug. Thank you!
